src/Movies.js

```diff
@@ -2,21 +2,24 @@ import React from 'react';
 import Poster from './Poster';
 import './Movies.css';
 
-const Movies = (props) => {
-const { movies } = props;
+const Movies = ({movies, findPoster}) => {
 
     const moviePosters = movies.map(movie => {
         return (
-            <Poster 
+            <Poster
                 image={movie.poster_path}
+                id={movie.id}
+                key={movie.id}
+                findPoster={findPoster}
             />
+
         )
     })
 
     return (
         <section>
             <h2 className='movies-section-title'>Movies</h2>
-            <article className='movie-posters-container'>
+            <article className={movies.length === 1 ? 'single-poster' : 'movie-posters-container'}>
```

Comment:
I'm glad we were able to use a ternary here!  It's so cool that you can use this to assign class names.